.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should not be committed. Isko .gitignore file may daldo. In fact, pura .vscode ka folder gitignore hoga

Reply:
done